doc/command_guidelines.md

```diff
@@ -22,19 +36,14 @@ If in doubt, ask!
 ## Command Naming and Behavior
 
 - Commands must follow a "[noun] [noun] [verb]" pattern
-- Multi-word subgroups should be hyphenated
-e.g. `foo-resource` instead of `fooresource`
-- All command names should contain a verb
-e.g. `account get-connection-string` instead of `account connection-string`
+- Multi-word subgroups should be hyphenated (e.g. `foo-resource` instead of `fooresource`)
+- All command names should contain a verb (e.g. `account get-connection-string` instead of `account connection-string`)
 - For commands which maintain child collections, the follow pairings are acceptable.
   1. `CREATE`/`DELETE` (same as top level resources)
   2. `ADD`/`REMOVE`
-- Avoid hyphenated command names when moving the commands into a subgroup would eliminate the need.
-e.g. `database show` and `database get` instead of `show-database` and `get-database`
-- If a command subgroup would only have a single command, move it into the parent command group and hyphenate the name. This is common for commands which exist only to pull down cataloging information.
-e.g. `database list-sku-definitions` instead of `database sku-definitions list`
-- In general, avoid command subgroups that have no commands. This often happens at the first level of a command branch.
-e.g. `keyvault create` instead of `keyvault vault create` (where `keyvault` only has subgroups and adds unnecessary depth to the tree).
+- Avoid hyphenated command names when moving the commands into a subgroup would eliminate the need (e.g. `database show` and `database get` instead of `show-database` and `get-database`).
+- If a command subgroup would only have a single command, move it into the parent command group and hyphenate the name. This is common for commands which exist only to pull down cataloging information (e.g. `database list-sku-definitions` instead of `database sku-definitions list`).
```

Comment:
I guess there is an exception to this when the group has one command but would almost certainly have more commands in the future

Reply:
Added.